From 38427e56b013da6c02ac7674c15bfde291fa0c31 Mon Sep 17 00:00:00 2001
From: Michal Suchanek <msuchanek@suse.de>
Date: Tue, 16 Jul 2019 13:14:05 +0200
Subject: [PATCH] KABI: remove unused hcall definition.

References: bsc#1140322 LTC#176270
Patch-mainline: no, kabi

H_SCM_PERFORMANCE_STATS is not used and it extends the bitmap in struct
kvm_arch. Remove it to preserve kABI

Signed-off-by: Michal Suchanek <msuchanek@suse.de>
---
 arch/powerpc/include/asm/hvcall.h | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/arch/powerpc/include/asm/hvcall.h b/arch/powerpc/include/asm/hvcall.h
index 37983f997577..b6645cf4675d 100644
--- a/arch/powerpc/include/asm/hvcall.h
+++ b/arch/powerpc/include/asm/hvcall.h
@@ -302,8 +302,7 @@
 #define H_SCM_QUERY_LOGICAL_MEM_BINDING 0x3F8
 #define H_SCM_UNBIND_ALL        0x3FC
 #define H_SCM_HEALTH            0x400
-#define H_SCM_PERFORMANCE_STATS 0x418
-#define MAX_HCALL_OPCODE	H_SCM_PERFORMANCE_STATS
+#define MAX_HCALL_OPCODE	H_SCM_HEALTH
 
 /* Scope args for H_SCM_UNBIND_ALL */
 #define H_UNBIND_SCOPE_ALL (0x1)
-- 
2.12.3

